src/PrimaryAuthController.js

```diff
@@ -141,8 +141,9 @@ function (Okta, PrimaryAuthForm, SocialAuth, PrimaryAuthModel, Util, BaseLoginCo
 
       // Add SocialAuth view only when the idps are configured. If configured, 'socialAuthPositionTop'
```

Comment:
Yea, i didn't see that. It doesn't fit in settings either. I will get rid of it.